api/src/foreign.rs

```diff
@@ -441,13 +441,13 @@ where
 	/// // If result okay, send to payer, who will apply the transaction via their
 	/// // owner API, then send back the slate
 	/// // ...
-	/// # let slate = Slate::blank(2);
+	/// # let mut slate = Slate::blank(2);
```

Comment:
Maybe we should have 2 methods instead of 1: `.tx()` and `.tx_mut()` to have a distinction between regular and mutable borrow. This would prevent having to add `mut`s and `&mut` in places where we don't really need them

Reply:
Yes, think that's fair. I was of two minds over this but there's probably a few too many things needed to be made mutable as a result of what's there currently